settings.yml

```diff
@@ -123,4 +123,4 @@ variables:
 #     value: 'OtherVersion'
 #   - name: preferredAzurePowerShellVersion
 #     value: '4.4.0'
-#
+#
```

Comment:
@karthikvenkat17 .. Everything is OK. Just one final comment. Please remove unnecessary commit to the `settings.yml`. Just a matter of restoring the file to the content from main.

Reply:
@ahmadabdalla Sorry that was an oversight on my part. Its restored now.